packages/editor/src/client/api/index.ts

```diff
@@ -342,17 +342,17 @@ export type EmbedBlock = {
   __typename?: 'EmbedBlock';
   url?: Maybe<Scalars['String']>;
   title?: Maybe<Scalars['String']>;
-  width?: Maybe<Scalars['Int']>;
-  height?: Maybe<Scalars['Int']>;
+  width?: Maybe<Scalars['String']>;
+  height?: Maybe<Scalars['String']>;
   styleCustom?: Maybe<Scalars['String']>;
   sandbox?: Maybe<Scalars['String']>;
 };
 
 export type EmbedBlockInput = {
   url?: Maybe<Scalars['String']>;
   title?: Maybe<Scalars['String']>;
-  width?: Maybe<Scalars['Int']>;
-  height?: Maybe<Scalars['Int']>;
+  width?: Maybe<Scalars['String']>;
```

Comment:
@tomaszdurka shouldn't we add here a mongodb migration that changes int to string? Or isn't this a problem if you have mixed types? I don't know mongodb enough.

Reply:
I don't think this needs a migration. @tomaszdurka what do you think?

Reply:
Ah, well it will come as Int, even though it should be String. It won't fail unless some string specific operations are made here.
I agree migration would help.

@penk85 can you write that quickly? @elias-summermatter can you help therE?

Reply:
@penk85 where you able to wrtie this migration?